### ISSUE TITLE
USCDR-14 DocumentReference resources returned conform to US Core R4 profiles test fails even if  DocumentReference.content.attachment.url is correct URL

### ISSUE THREAD

**Issue by ykulkarnidia** [onc-healthit/inferno#449]
Execute the USCore310DocumentreferenceSequence from uscore_v3.1.0 test suite.
Test case  - USCDR-14 DocumentReference resources returned conform to US Core R4 profiles
```
 X fail - USCDR-14 DocumentReference resources returned conform to US Core R4 profiles
    Message: DocumentReference/5e8cd2544ee1c10006d58936: DocumentReference.content.attachment.url is not a valid url: 'http://fhir:3447/4_0_0/Binary/5e8cd2544ee1c10006d58936?type=document'<br/>
DocumentReference/5e8cd2544ee1c10006d58936: DocumentReference.content.attachment.url did not match one of the valid data types: ["url"]
    200 GET http://fhir:3447/4_0_0/DocumentReference/5e8cd2544ee1c10006d58936
```

Here is the actual data for content element -
```
    "content": [
        {
            "attachment": {
                "contentType": "text/xml",
                "url": "http://fhir:3447/4_0_0/Binary/5e8cd2544ee1c10006d58936?type=document",
                "title": "ccd for fhir demo"
            }
        }
    ],
```

**Your environment**
* Edition of inferno (Community or Program): Community
* Version of inferno: Version of inferno: 2.9.0
* Browser and version(s) is the bug present on?: Tested via batch file. 

It seems like the data types  /lib/fhir_models/fhir_ext/structure_definition.rb are still pointing to older version of fhir spec( http://hl7.org/fhir/2015May/datatypes.html). They have not been updated 4.0.0

Here some additional info
```
INFO | Inferno | GETTING: http://fhir:3447/4_0_0/DocumentReference/5e8cd2544ee1c10006d58936
WARN | Inferno | Unable to check http://fhir:3447/4_0_0/Binary/5e8cd2544ee1c10006d58936?type=document for datatype url
WARN | Inferno | Unable to check http://fhir:3447/4_0_0/Binary/5e8cd2544ee1c10006d58936?type=document for datatype url
ERROR | Inferno | AssertionException: DocumentReference/5e8cd2544ee1c10006d58936: DocumentReference.content.attachment.url is not a valid url: 'http://fhir:3447/4_0_0/Binary/5e8cd2544ee1c10006d58936?type=document'<br/>
DocumentReference/5e8cd2544ee1c10006d58936: DocumentReference.content.attachment.url did not match one of the valid data types: ["url"]
INFO | Inferno | Finished Test: USCDR-14 [fail]
```

**Comment by yunwwang** [comment]
Hello ykulkarnidia, thank you for reaching to Inferno. For the context, I think you run the test from your forked branch. Is that correct? Did you run using batch file? What is the validator setting in your config.yml? Thank you.

**Comment by ykulkarnidia** [comment]
Hi yunwang,

I did execute using a batch file but not via fork. I created a fork to ignore some of cases that were failing for us. We are using resource_validator: internal in our config.yml

**Comment by yunwwang** [comment]
Hello ykulkarnidia: 

Thank you for the confirmation. We have just updated fhir_model to 4.1.0. Hopefully that will solve this issue.

**Comment by yunwwang** [comment]
Close this issue as code fix is released in v2.10.0. PR: #481
